src/Lidar.cc

```diff
@@ -166,6 +169,17 @@ bool Lidar::Load(const sdf::Sensor &_sdf)
     }
   }
 
+  sdf::ElementPtr element = _sdf.Element();
+  // Get the background color, if specified.
```

Comment:
Is this a copy/paste error?

Reply:
solved https://github.com/ignitionrobotics/ign-sensors/pull/195/commits/9a56643f76ec701f1df54e4da177b035b95c674f, thank you!